app/src/main/AndroidManifest.xml

```diff
@@ -24,7 +35,7 @@
     android:label="@string/app_name"
     android:supportsRtl="true"
     android:theme="@style/Clinic.V2.Theme"
-    tools:ignore="AllowBackup,GoogleAppIndexingWarning"
+    tools:ignore="AllowBackup,GoogleAppIndexingWarning, LockedOrientationActivity"
```

Comment:
What would `LockedOrientationActivity` do?

Reply:
We are basically ignoring a lint error for `android:screenOrientation` being locked to portrait.